tests/osaka/eip7692_eof_v1/eip4750_functions/test_callf_execution.py

```diff
@@ -144,90 +119,80 @@ def test_callf_stack_size_1024_at_push(
                 ),
             ],
         ),
-        container_post=Account(storage={slot_code_worked: value_code_worked}),
-    )
-
-
-def test_callf_stack_overflow(
-    eof_state_test: EOFStateTestFiller,
-):
-    """Test stack overflowing 1024 items in called function"""
-    eof_state_test(
-        data=Container(
+        Container(
+            name="nested_with_inputs_at_push0",
             sections=[
                 Section.Code(
-                    code=Op.PUSH0 * 1023
+                    code=Op.PUSH0 * 1022
                     + Op.CALLF[1]
-                    + Op.POP * 1023
+                    + Op.POP * 1022
                     + Op.SSTORE(slot_code_worked, value_code_worked)
```

Comment:
there is Storage iterator class from Mario, check it out

Reply:
I've never used it at the global scope either, so it's fine to leave these as is for the time being.